agent/bench-scripts/tests/pbench-fio/test-22.txt

```diff
@@ -1,5 +1,5 @@
 +++ Running test-22 pbench-fio
-[error][1900-01-01T00:00:00.000000] [pbench-fio] fio executable not found
+[error][1900-01-01T00:00:00.000000] [pbench-fio] fio failed reporting its version: '/var/tmp/pbench-test-bench/opt/pbench-agent/bench-scripts/pbench-fio: line 476: fio: command not found'
```

Comment:
We like having the "script name" including the full directory path?  (I think I'm OK with it, but I don't think it's uniform practice.)